fulfillment-service/it/it_volume_lifecycle_test.go

```
@@ -0,0 +1,496 @@
+				}.Build())
+			Expect(err).ToNot(HaveOccurred())
+			volObj := getResp.GetObject()
+			volObj.GetSpec().SetStorageTier("different-tier")
+			_, err = volumesClient.Update(ctx,
+				privatev1.VolumesUpdateRequest_builder{
+					Object: volObj,
+				}.Build())
+			Expect(err).To(HaveOccurred())
+		})
+	})
+})
```

golangci-lint (2.13.2)

[error] File is not properly formatted

(gofmt)